db.go

```diff
@@ -109,6 +110,10 @@ func (db *DB) Insert(what string, data interface{}) (interface{}, error) {
 	return db.send("insert", what, data)
 }
 
+func (db *DB) LiveNotifications(id string) (chan rpc.RPCResponse, error) {
```

Comment:
people cannot use internal packages maybe we can move rpc to pkg

Reply:
I am giving high-level feedback here without digging too much into the implementation.

The driver should not expose internal details at all to users. This should be `Notification` if it is user facing.

Reply:
I agree but if we want people to use `rpc.RPCResponse` we should move them to `pkg` folder maybe we should separate the files but I'm not sure if that good idea either.

Reply:
We don't want people to use rpc.RPCResponse. This is because, down the line, it might change. And we don't want users to know about that change when and if it does happen.

Reply:
I agree that if we did, it would be a separate package though